src/pages/PersonalSiteTemplateEspanol.page-meta.xml

```diff
@@ -3,5 +3,5 @@
     <apiVersion>40.0</apiVersion>
     <availableInTouch>false</availableInTouch>
     <confirmationTokenRequired>false</confirmationTokenRequired>
-    <label>PersonalSiteTemplateEspañol</label>
+    <label>PersonalSiteTemplateEspanol</label>
```

Comment:
is there a reason the accented ñ was removed from this?

Reply:
My guess is an issue with CumulusCI. I'll try to take a look this week.

Reply:
I can remove the change for now, and we can fix later if it is something that has to be addressed but as an fyi you may need to change your local copy to deploy.